features/volledigeNaam.feature

```diff
@@ -80,25 +71,22 @@ Rule: een predicaat wordt opgenomen voor de voornamen met de omschrijving horend
 
     Voorbeelden:
     | burgerservicenummer | geslacht | voornamen            | adellijkeTitelPredicaat | voorvoegsel | geslachtsnaam | volledigeNaam                               |
-    | 999994669           | V        | Żáïŀëñøŕ Åłéèç Đëļŧå | JV                      | 'S          | Streeveld     | jonkvrouw Żáïŀëñøŕ Åłéèç Đëļŧå 'S Streeveld |
-    | 555550001           | V        | Jo Anne              | JV                      | van         | Hoogh         | jonkvrouw Jo Anne van Hoogh                 |
-    | 555550002           | M        | Jo Anne              | JV                      | van         | Hoogh         | jonkheer Jo Anne van Hoogh                  |
-    | 555550003           | O        | Jo Anne              | JV                      | van         | Hoogh         | jonkvrouw Jo Anne van Hoogh                 |
-    | 555550004           | V        | Jo Anne              | JH                      | van         | Hoogh         | jonkvrouw Jo Anne van Hoogh                 |
-    | 555550005           | M        | Jo Anne              | JH                      | van         | Hoogh         | jonkheer Jo Anne van Hoogh                  |
-    | 555550006           | O        | Jo Anne              | JH                      | van         | Hoogh         | jonkheer Jo Anne van Hoogh                  |
+    | 000000541           | V        | Żáïŀëñøŕ Åłéèç Đëļŧå | JV                      | 'S          | Streeveld     | jonkvrouw Żáïŀëñøŕ Åłéèç Đëļŧå 'S Streeveld |
+    | 000000553           | V        | Jo Anne              | JV                      | van         | Hoogh         | jonkvrouw Jo Anne van Hoogh                 |
+    | 000000565           | M        | Jo Anne              | JV                      | van         | Hoogh         | jonkheer Jo Anne van Hoogh                  |
+    | 000000577           | O        | Jo Anne              | JV                      | van         | Hoogh         | jonkvrouw Jo Anne van Hoogh                 |
+    | 000000589           | V        | Jo Anne              | JH                      | van         | Hoogh         | jonkvrouw Jo Anne van Hoogh                 |
+    | 000000590           | M        | Jo Anne              | JH                      | van         | Hoogh         | jonkheer Jo Anne van Hoogh                  |
+    | 000000607           | O        | Jo Anne              | JH                      | van         | Hoogh         | jonkheer Jo Anne van Hoogh                  |
     
 
 Rule: een adellijke titel wordt opgenomen tussen de voornamen en het voorvoegsel met de omschrijving horend bij de waarde in tabel 38 in de vorm die hoort bij het geslacht en geschreven in kleine letters
   - wanneer er geen vorm is die hoort bij de adellijke titel (geslacht "O" of titel "R" met geslacht "V"), wordt de omschrijving horend bij de waarde in tabel 38 gebruikt
```

Comment:
klopt niet, je krijgt geen titel/predicaat

Reply:
Besproken in werkoverleg